src/commands/download_file.ts

```diff
@@ -0,0 +1,37 @@
+import { EID } from 'ardrive-core-js';
+import { resolve as resolvePath } from 'path';
+import { cliArDriveAnonymousFactory, cliArDriveFactory } from '../index';
+import { CLICommand, ParametersHelper } from '../CLICommand';
+import { CLIAction } from '../CLICommand/action';
+import {
+	DriveIdParameter,
+	DrivePrivacyParameters,
+	FileIdParameter,
+	LocalFilePathParameter
+} from '../parameter_declarations';
+
+new CLICommand({
+	name: 'download-file',
+	parameters: [FileIdParameter, LocalFilePathParameter, DriveIdParameter, ...DrivePrivacyParameters],
```

Comment:
Make this non-required here or remove the DriveID param altogether

```suggestion
	parameters: [
		FileIdParameter,
		LocalFilePathParameter,
		{ name: DriveIdParameter, required: false },
		...DrivePrivacyParameters
	],
```

Reply:
And probably switch `LocalFilePathParameter` for a new `DestFolderPathParameter` since that is the behavior its currently using